frontend/src/components/filter/FilterStateSelect.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState, useEffect, useContext } from 'react';
+import PropTypes from 'prop-types';
+import FilterSelect from './FilterSelect';
+import UserContext from '../../UserContext';
+import { getStateCodes } from '../../fetchers/users';
+import { allRegionsUserHasPermissionTo } from '../../permissions';
+
+const ALL_STATES = [
```

Comment:
A comment explaining these are the states for regions in region order would be helpful.